test/reductions/grid_search/test_grid_search_arguments.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import logging
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.linear_model import LogisticRegression, LinearRegression
+
+from fairlearn.metrics import DemographicParity, BoundedGroupLoss
+from fairlearn.reductions import GridSearch
+from fairlearn.reductions.grid_search.simple_quality_metrics import SimpleClassificationQualityMetric, SimpleRegressionQualityMetric  # noqa: E501
+
+
+def identity(X):
+    return X
+
+
+def pandasdf(X):
```

Comment:
you don't need all these functions. You can provide them in the list below directly
```
X_transform = [lambda X: X, pd.DataFrame]

Reply:
I think it's clearer to have them all as functions grouped at the top; it is something of a matter of taste, though.